src/api/student.js

```diff
@@ -25,17 +25,16 @@ app.post('/register', (req, res) => {
         res.failureJson(e.message);
         return;
     }
-    let salt = createToken();
     let confirmStudentEmailToken = createToken();
     let confirmBackupEmailToken = createToken();
-    let hashedPassword = saltAndHash(password, salt);
-    publish('NEW_STUDENT', {
-        studentEmail,
-        backupEmail,
-        confirmStudentEmailToken,
-        confirmBackupEmailToken,
-        hashedPassword,
-        salt
```

Comment:
Yes, but I think it only has the hash. I don't think it really matters, however.

https://github.com/P-H-C/phc-string-format/blob/master/phc-sf-spec.md